coc/http.py

```diff
@@ -66,15 +67,26 @@ def __init__(self, method, path, kwargs: dict=None, api_page=False):
 
 
 class HTTPClient:
-    def __init__(self, client, token, loop=None, *, email, password, update_tokens=False):
+    def __init__(self, client, tokens, loop=None, *, email, password, update_tokens=False):
         self.client = client
-        self.token = token
         self.loop = asyncio.get_event_loop() if loop is None else loop
         self.__session = None
         self.email = email
         self.password = password
         self.update_tokens = update_tokens
         self.loop.run_until_complete(self.login())
+        
+        if not tokens:
+            response_dict, session = await self.login_to_site(self.email, self.password)
+            cookies = self.create_cookies(response_dict, session)
+            self._tokens = await self.find_site_tokens(cookies)['keys']
+        else:
+            self._tokens = tokens
+            
+            self._tokens = [token['key'] for token in current_tokens]
```

Comment:
whats this do? should it be inside the `if not tokens:` block?

Reply:
Basically if tokens is none then we get them from the email/pass

Reply:
Oh just read that, yes it should be oops my bad